sign-in.txt

```diff
@@ -1,7 +1,9 @@
 ﻿This is an incredibly useful text document.
 //TODO: Be sure you have created a feature branch from develop 
 //TODO: Please put your name and github username AS SHOWN below ... 
-  ... then commit, pull origin develop, merge develop into your branch, push origin feature, and review your pull request
-  
+<<<<<<< HEAD
+//       ... then commit and create pull request to develop
```

Comment:
Willie, please leave my extended comment above, rather than your shorter one

Reply:
I have corrected the issue.